integration-test.py

```diff
@@ -159,21 +159,57 @@ def match_distance(actual, expected):
     return (distance, misses)
 
 
-@contextmanager
-def features_in_tile_layer(z, x, y, layer):
+class GatewayTimeout(Exception):
+    def __init__(self, message):
+        self.message = message
```

Comment:
Thanks! I didn't know about the `pass` thing. I guess that just means it'll use `Exception`'s `__init__` method? Fixed in e03f14a.